tools/format/format.conf

```diff
@@ -405,6 +405,28 @@ emit:chance
 ^mission_definition:dialogue:success_lie
 ^mission_definition:dialogue:failure
 
+# Bionics
+^bionic
+^bionic:id
+^bionic:type
+^bionic:name
+^bionic://
+^bionic:capacity
+^bionic:description
+^bionic:occupied_bodyparts
```

Comment:
I think so, it cuts down on vertical lines a lot which is generally a good thing for readability